version.py

```diff
@@ -40,18 +40,27 @@
 
 
 def call_git_describe(prefix='', cwd='.'):
-    cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    version_name_cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    last_tag_cmd = 'git describe --tags --abbrev=0'
```

Comment:
Why not `version_cmd` for consistency with `version` variable below?

Reply:
Done